packages/core/sds/src/components/Skeleton/Paragraphs.tsx

```diff
@@ -8,18 +8,16 @@ export interface SkeletonParagraphsProps extends HTMLAttributes<HTMLDivElement>
   count?: number;
 }
 
-export const SkeletonParagraphs = forwardRef<HTMLDivElement, SkeletonParagraphsProps>(
-  (props: SkeletonParagraphsProps) => {
-    const { count = 3 } = props;
+export const SkeletonParagraphs = forwardRef<HTMLDivElement, SkeletonParagraphsProps>((props, ref) => {
+  const { count = 3 } = props;
```

Comment:
```suggestion
  const { count = 3, ...restProps } = props;
```

요거 해서 아래 가장 상위 div에 넘겨주는 것도 필요할 거 같아요 ..ㅋㅋㅋ 다 빼먹었네요 제가

Reply:
https://github.com/depromeet/15th-team3-FE/pull/108/commits/ec5f98d7859037604eb57eaaa7e5220c9c012c71